BlockchainSdk/WalletManagers/Ethereum/ERC20TokenMethods/AllowanceERC20TokenMethod.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AllowanceERC20TokenMethod.swift
+//  BlockchainSdk
+//
+//  Created by Sergey Balashov on 18.09.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+public struct AllowanceERC20TokenMethod: SmartContractMethod {
+    public let spender: String
+    public let owner: String
+    
+    public init(owner: String, spender: String) {
+        self.owner = owner
+        self.spender = spender
+    }
+    
+    public var prefix: String { "0xdd62ed3e" }
+    public var data: Data {
```

Comment:
Слиплось везде. И наверно лучше перед конструктором везде положить эти переменные

```suggestion
    public var prefix: String { "0xdd62ed3e" }
    
    public var data: Data {
```

Reply:
убрал в `extension` ✅